app/logic/Calendar/Event.ts

```diff
@@ -230,6 +236,12 @@ export class Event extends Observable {
     this.unedited = null;
   }
 
+  createMeeting() {
+    assert(this.participants.isEmpty, "This is already a meeting");
+    assert(this.myParticipation <= InvitationResponse.Organizer, "Incoming invitation should already have participants");
```

Comment:
```suggestion
    assert(this.myParticipation == InvitationResponse.Organizer, "Incoming invitation should already have participants");
```

Please do not use `<=` et al at all for enums.

Use `||` or `!` or `IncomingStates.includes(this.myParticipation)` or similar as necessary.